docs/source/examples/workspaces/dummy-python-hook/linchpin.conf

```diff
@@ -0,0 +1,326 @@
+# This file is a well-documented, and commented out (mostly) file, which
+# covers the configuration options available in LinchPin
+#
+# Used to override default configuration settings for LinchPin
+# Defaults exist in linchpin/linchpin.constants file
+#
+# Uncommented options enable features found in v1.5.1 or newer and
+# can be turned off by commenting them out.
+#
+# structured in INI style
+# use %% to allow code interpolation
+# use % to use config interpolation
+#
+
+[DEFAULT]
+# name of the python package (Redundant, but easier than programmatically
+# obtaining the value. It's very unlikely to change.)
+pkg = linchpin
+
+# Useful for storing the RunDB or other components like global credentials
+# travis-ci doesn't like ~/.config/linchpin, use /tmp
+#default_config_path = ~/.config/linchpin
+
+# When creating an provider not already included in LinchPin, this path
+# extends where LinchPin will look to run the appropriate playbooks
+#external_providers_path = %(default_config_path)s/linchpin-x
+
+# When adding anything to the lp section, it should be general for
+# the entire application.
+[lp]
+
+# load custom ansible modules from here
+#module_folder = library
+
+# rundb tracks provisioning transactions
+# If you add a new one, rundb/drivers.py needs to be updated to match
+# rundb_conn is the location of the run database.
+# A common reason to move it is to use the rundb centrally across
+# the entire system, or in a central db on a shared filesystem.
+# System-wide RunDB: rundb_conn = ~/.config/linchpin/rundb-::mac::.json
+rundb_conn = {{ workspace }}/.rundb/rundb-::mac::.json
+#rundb_conn = ~/.config/linchpin/rundb-::mac::.json
+
+# name the type of Run database. Currently only TinyRunDB exists
+#rundb_type = TinyRunDB
+
+# How to connect to the RunDB, if it's on a separate server,
+# it may be tcp or ssh
+rundb_conn_type = file
```

Comment:
@14rcole : cant we symlink the linchpin.conf ? It would definitely de-duplicate lot of files 
same comment

Reply:
Yes, good catch.  I did this before I knew to change linchpin.constants.  I would argue that we don't really need it at all anymore